docs/troubleshooting/README.md

```diff
@@ -72,3 +72,19 @@ If they are then you are using the syntax prior to [LunarVim#1813](https://githu
 -- this is the correct syntax since 3dd60bd
 vim.list_extend(lvim.lsp.override, { "jsonls" })
 ```
+
+## Running LunarVim on M1 mac reports error `mach-o, but wrong architecture`
+
+On start up, lunarvim reports errors such as below with several plugins.
+
+``` bash
+no suitable image found.  Did find:
+    $HOME/.local/share/nvim/site/pack/packer/start/nvim-treesitter/parser/kotlin.so: mach-o, but wrong architecture
+    $HOME/.local/share/nvim/site/pack/packer/start/nvim-treesitter/parser/kotlin.so: mach-o, but wrong architecture
+```
+
+Lunarvim does not support M1 mac natively. This can be fixed by installing
+homebrew and neovim through the x86_64 architecture platfomr location
+`/usr/local`. Doing so will run neovim and homebrew through the x86_64
+architecture. This only works provided you have Rosetta installed. 
```

Comment:
I just got it confirmed on discord that you can have a native `mac-o` installation of Neovim through macports, so this not completely accurate.

```suggestion
```

Reply:
Ok, do you mind detailing here how those individuals did that? I tried mostly everything I could to install `brew` and `neovim` and its dependencies through `/opt/homebrew` but it always led to the issue linked above.

Reply:
I just used macports which installed neovim as a native M1 application without any tweaks.